rviz_default_plugins/CMakeLists.txt

```diff
@@ -117,9 +120,11 @@ target_link_libraries(rviz_default_plugins
   rviz_rendering::rviz_rendering
   rviz_common::rviz_common
   resource_retriever::resource_retriever
+  laser_geometry::laser_geometry
```

Comment:
nitpick: alphabetical order unless the library order matters (I don't think it does)